dependencies.yaml

```diff
@@ -91,6 +91,7 @@ dependencies:
         packages:
           - c-compiler
           - cxx-compiler
+          - gmock>=1.13.0
```

Comment:
It surprises me that gtest is not in this list, only gmock. Typically we use those together.

Reply:
You are 100% correct, this should be both.